src/allowedDataTypes.ts

```diff
@@ -0,0 +1,21 @@
+const allowedDataTypes = [
```

Comment:
I'm pretty sure we don't yet support around of these data types like time or wikibase-item, they all need to be removed and only things that have "value type" of string should stay. I will find the list for you now.

Reply:
The list: https://github.com/wikimedia/Wikibase/blob/master/lib/WikibaseLib.datatypes.php#L18

Reply:
oh, ok. That was not made clear in the ticket, I can leave them commented out to be added later easily.